shared/oval/sshd_limit_user_access.xml

```diff
@@ -0,0 +1,28 @@
+<def-group>
```

Comment:
this directory does not seem to exist yet? https://github.com/OpenSCAP/scap-security-guide/tree/master/shared/templates/static